src/storage/conversion/QuadToRdfConverter.ts

```diff
@@ -17,12 +17,21 @@ import { TypedRepresentationConverter } from './TypedRepresentationConverter';
  * Converts `internal/quads` to most major RDF serializations.
  */
 export class QuadToRdfConverter extends TypedRepresentationConverter {
+  private readonly outputPreferences?: ValuePreferences;
+
+  public constructor(options: { outputPreferences?: Record<string, number> } = {}) {
+    super();
+    if (Object.keys(options.outputPreferences ?? {}).length > 0) {
+      this.outputPreferences = { ...options.outputPreferences };
+    }
+  }
+
   public async getInputTypes(): Promise<ValuePreferences> {
     return { [INTERNAL_QUADS]: 1 };
   }
 
   public async getOutputTypes(): Promise<ValuePreferences> {
-    return rdfSerializer.getContentTypesPrioritized();
+    return this.outputPreferences ?? rdfSerializer.getContentTypesPrioritized();
```

Comment:
Everything about this feels like it should be more generalized for converters in general and not just this one. What I'm thinking of is some sort of abstract class that handles this behaviour and has specific abstract functions that the converter needs to implement to support this. But this would probably be quite a bit more complex than this solution so might be something for later.

Reply:
Perhaps indeed; but could be overkill for now.

Reply:
Done in ecc21d467e85236cecc650f439510f16b9aebde5